pkg/model/deployment_trace.go

```diff
@@ -14,6 +14,12 @@
 
 package model
 
+const (
+	// The key to store the commit hash that triggers the event (in EventWatcher flow).
+	// It will be used as key to store the commit hash as metadata in the commit boy.
```

Comment:
[nit] to simplify + fix typo
```suggestion
	// The key to store the commit hash that triggers the event (in EventWatcher flow) as metadata in the commit body.
```